samples/simple-android/src/main/java/com/fortysevendeg/mvessel/test/ui/MainActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.fortysevendeg.mvessel.test.ui;
+
+import android.app.Activity;
+import android.app.LoaderManager;
+import android.content.Context;
+import android.content.Loader;
+import android.database.Cursor;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CursorAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+import com.fortysevendeg.mvessel.test.R;
+import com.fortysevendeg.mvessel.test.db.ContactsCursorLoader;
+import com.fortysevendeg.mvessel.test.db.ContactsOpenHelper;
+
+public class MainActivity extends Activity implements LoaderManager.LoaderCallbacks<Cursor> {
```

Comment:
Does it make sense if we create another module like `simple-android` where you would show how to use mvessel since Android Scala App?